apps/projects/app/components/Content/IssueDetail/components/Detail.js

```diff
@@ -210,8 +217,17 @@ const Detail = ({
               </Text.Block>
             </div>
             <div style={{ ...column, flex: 0, alignItems: 'flex-end' }}>
-              <DropDownButton enabled />
-              {/* <Text>$500 USD</Text> */}
+              <DropDownButton enabled>
```

Comment:
The dropdowns both here and on the issue detail card stopped rendering on click after the app crashed. I think it's not a coincidence, but it's worth noting